src/ngsa/app/DataAccessLayer/InMemoryDal.cs

```diff
@@ -9,12 +9,13 @@
 using CSE.NextGenSymmetricApp.Model;
 using Microsoft.Azure.Cosmos;
 using Microsoft.Extensions.DependencyInjection;
+
+// TODO - convert to system.text.json?
 using Newtonsoft.Json;
```

Comment:
That's a great idea - I'll add to the board to discuss at triage.

Note that for this instance, this is resolved in another branch, so I removed the TODO as it will be fixed when we merge the branches.